x-pack/metricbeat/module/gcp/metrics/metricset.go

```diff
@@ -103,6 +103,7 @@ type config struct {
 	Zone                string   `config:"zone"`
 	Region              string   `config:"region"`
 	Regions             []string `config:"regions"`
+	ServiceLabel        string   `config:"servicelabel"`
```

Comment:
I wonder if `LocationLabel (location_label)` would be a better name for this config param. `ServiceLabel` feels too generic it could refer to any label, but here we’re specifically referring to region/zone/location labels :thinking: